molecule/plaintext-rhel-customrepo/prepare.yml

```diff
@@ -0,0 +1,41 @@
+---
+- name: Prepare
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  tasks:
+    - set_fact:
+        ansible_os_family: "RedHat"
+        ansible_distribution_major_version: 7
+    - import_role:
+        name: variables
+    - debug:
+        msg:
+          - "Confluent Client Repo Version -> {{ confluent_common_repository_redhat_release_version }}"
+          - "OS Family -> {{ ansible_os_family }}"
+    - name: Create custom repo file
+      copy:
+        dest: "{{ custom_yum_repofile_filepath }}"
+        content: |
+          [Confluent.custom]
+          async = 1
+          baseurl = "https://packages.confluent.io/rpm/{{confluent_repo_version}}"
+          enabled = 1
+          gpgcheck = 1
+          gpgkey = "https://packages.confluent.io/rpm/{{confluent_repo_version}}/archive.key"
+          name = Custom Confluent repository
+          [Confluent.clients.custom]
+          async = 1
+          baseurl = "https://packages.confluent.io/clients/rpm/centos/{{confluent_common_repository_redhat_release_version}}/x86_64"
```

Comment:
If we're having x86_64, would that mean that this will only run with this arch on target nodes? 
In that case, should we add a note in the molecule.yml file so that people know where to run/not run?
Or, if there's a way to resolve architecture somehow, that would be awesome too.

Reply:
For confluent clients, confluent only have repo for x86_64 and this client works for both type of architectures.